package.json

```diff
@@ -28,18 +28,20 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@babel/runtime": "7.22.15",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@material-table/core": "^4.3.11",
     "@mov-ai/mov-fe-lib-code-editor": "^1.0.8-2",
     "@remixproject/engine": "^0.3.19",
     "@remixproject/engine-web": "^0.3.19",
+    "clsx": "^1.1.1",
```

Comment:
We didn't need this before, why do we need it now?

Reply:
Because we are using it: ![2023-09-25-095518_1920x1080_scrot](https://github.com/MOV-AI/frontend-npm-lib-ide/assets/4276265/548fffaa-1e76-4b76-94c1-fc4d3f7c9fc2)

I don't know why it didn't complain before.

Reply:
I removed its only usage